phoenix-core-client/src/main/java/org/apache/phoenix/jdbc/HighAvailabilityPolicy.java

```diff
@@ -47,6 +48,23 @@ public Connection provide(HighAvailabilityGroup haGroup, Properties info,
             return new FailoverPhoenixConnection(context);
         }
 
+        /**
+         * Cluster Role Transitions for Failover High Availability Policy, Here we are trying to
+         * close connections late to allow existing reads to continue during Failover.
+         * ACTIVE --> ACTIVE_TO_STANDBY (Doing Nothing as we are in process of moving the current
```

Comment:
how about state transition from ACTIVE_TO_STANDBY --> ACTIVE? We will need this to rollback state change from ACTIVE --> ACTIVE_TO_STANDBY.

Reply:
ACTIVE --> ACTIVE_TO_STANDBY is not doing anything here, as we just want to block writes and nothing else, So if we go to ACTIVE_TO_STANDBY then rolling back should be same (nothing required), 
I see I missed commenting that will add the comment :v: